referral_module/api/tests/test_serializers_api.py

```diff
@@ -0,0 +1,109 @@
+from django.test import TestCase
+from collections import OrderedDict
+
+from django.contrib.auth.models import User
+from django.urls import reverse
+
+from rest_framework.test import APIRequestFactory
+
+from referral_module.tests import factories
+from referral_module.api import serializers
+from referral_module import models
+
+
+class SerializersApiTestCase(TestCase):
+    def setUp(self):
+        self.user = User.objects.create_user('usertest', 'user@mail.com', 'liorlior')
+        self.client.login(username='usertest', password='liorlior')
+        self.campaigns = []
+        for i in range(0, 5):
+            self.campaigns.append(factories.CampaignFactory(
+                description='some some'
+            ))
+
+    def test_campaigns_list_serializer(self):
+        queryset = models.Campaign.objects.all()
+        serializer = serializers.CampaignSerializer(queryset, many=True)
+
+        # Check that 5 campaigns was serialized
+        self.assertEqual(len(serializer.data), 5)
+
+        campaigns_data_list = []
+        for campaign in self.campaigns:
+
+            campaign_dict = OrderedDict(
+                [
+                    ('id', campaign.id), ('key', campaign.key), ('description', campaign.description),
+                    ('is_active', campaign.is_active), ('created', campaign.created.strftime('%Y-%m-%dT%H:%M:%S.%fZ'))
```

Comment:
For the datetime formatting would be nice to use restframework settings instead of hardcoding it `%Y-%m-%dT%H:%M:%S.%fZ`